infra/recipes/k8s/environments/azure/sandbox/tables/values.yaml

```diff
@@ -0,0 +1,34 @@
+namespace: "default"
+
+openhouseClusterTag: "openhouse"
+
+clusterPropertiesPath: "config/azure/sandbox/cluster.yaml"
+
+tablesService:
+  replicaCount: 1
+  image:
+    repository: "containerregistryopenhouse.azurecr.io/openhouse-tables-service"
+    tag: "latest"
+    pullPolicy: IfNotPresent
+  service:
+    type: NodePort
+    port: 8080
+  port: 8080
+  resources:
+    requests:
+      cpu: 1
+      memory: 1G
+    limits:
+      cpu: 2
```

Comment:
Should use 1 cpu to avoid `cpu insufficient` error right?

Reply:
Yes so it is a tight bound